app/services/cards.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import config from '../../config';
+import { externalApiError } from '../errors';
+import logger from '../logger';
+import { CardsInfo, Cards } from '../types/cards';
+
+export const BASE_URL = config.heartstone.baseURL as string;
+export const API_KEY = config.heartstone.APIKey as string;
+
+const client = axios.create({
+  baseURL: BASE_URL,
+  responseType: 'json',
+  headers: {
+    'x-rapidapi-host': BASE_URL,
+    'x-rapidapi-key': API_KEY
+  }
+});
+
+export async function getCardsInfo(): Promise<CardsInfo[]> {
```

Comment:
I think that this querying info should return a single object instead of the entire list since it would not have difference with the getAllCards function.